### PR TITLE
VACMS-10277: enhance facility services reports with additional fields

### LINKED ISSUE #10277: Enhance facility services report with additional fields.
## User Story or Problem Statement

Slack thread: https://dsva.slack.com/archives/C0FQSS30V/p1660592074016579

As a user with access to the VAMC Facility Health Services report (view), I would like these reports (see list below) and the corresponding CSV display to be updated as follows:

Views needing facility ids: 
- [x] /admin/content/facilities/vamc-facility-health-services
- [x] /admin/content/facilities/vet-center-services
- [x] /admin/content/facilities/vamc-facility-non-clinical-services

For each view listed, make sure:
- [x] Add field - Facility ID
- [x] Add field - Moderation State (published/draft/archive/etc)
- [x] Modify Facility Name field in CSV display (should be returned as plain text instead of a link)
- [x] Add Moderation State as a filter

### CMS Team
Please check the team(s) that will do this work.

- [ ] `Program`
- [ ] `Platform CMS Team`
- [ ] `Sitewide Crew`
- [ ] `⭐️ Sitewide CMS`
- [ ] `⭐️ Public Websites`
- [x] `⭐️ Facilities`
- [ ] `⭐️ User support`

### PR DESCRIPTION
**Pull request by JayDarnell**
## Description

Closes #10277 

## Testing done

Manual testing

## Screenshots

Screen Shot 2022-08-22 at 5 42 12 PM

Screen Shot 2022-08-22 at 5 41 06 PM

Screen Shot 2022-08-22 at 5 42 31 PM

## QA steps

- [x] Visit: /admin/content/facilities/vamc-facility-health-services
- [x] Verify the Facility Locator ID, and Moderation state fields have been added
- [x] Verify the Moderation state filter has been added and works
- [x] Filter by Draft moderation state
- [x] Click the Download CSV
- [x] Open the CSV file and verify the Facility field is just the title (no html) and that the Facility Locator ID and Moderation state fields have been added
- [x] Visit: /admin/content/facilities/vet-center-services
- [x] Verify the Vet Center, Facility Locator ID, and Moderation state fields have been added
- [x] Verify the Moderation state filter has been added and works
- [x] Filter by Draft moderation state
- [x] Click the Download CSV
- [x] Open the CSV file and verify the Vet Center field is just the title (no html) and that the Facility Locator ID and Moderation state fields have been added
- [x] Visit: /admin/content/facilities/vamc-facility-non-clinical-services
- [x] Verify the Facility Locator ID, and Moderation state fields have been added
- [x] Verify the Moderation state filter has been added and works
- [x] Filter by Draft moderation state
- [x] Click the Download CSV
- [x] Open the CSV file and verify the Facility field is just the title (no html) and that the Facility Locator ID and Moderation state fields have been added

### Select Team for PR review

- [ ] `Platform CMS Team`
- [ ] `Sitewide program`
- [ ] `⭐️ Sitewide CMS`
- [ ] `⭐️ Public websites`
- [x] `⭐️ Facilities`
- [ ] `⭐️ User support`

### CODE REVIEW

**Comment by omahane** [comment]
@JayDarnell the /admin/structure/views/view/facility_services/edit/vamc_facility_health_services_export
has HTML in
- Title
- Section
- Updated

It also has "edit_node" which we do not need for the CSV.
Screen Shot 2022-08-23 at 2 53 04 PM
[vamc-facility-health-services.csv](https://github.com/department-of-veterans-affairs/va.gov-cms/files/9410276/vamc-facility-health-services.csv)

**Comment by omahane** [comment]
There's HTML in this export, too:
/admin/structure/views/view/facility_services/edit/vet_center_services_export

**Comment by omahane** [comment]
There's HTML in this export, too:
/admin/structure/views/view/facility_services/edit/vamc_facility_non_clinical_services_export

**Comment by omahane** [comment]
Other than the HTML and the "edit_node" field in the export, everything looks good.

**Review by omahane — DISMISSED** [review]
Render the export fields without HTML and remove the "edit_node" from them.

**Comment by swirtSJW** [comment]
Good catch @omahane

**Comment by JayDarnell** [comment]
@omahane @swirtSJW updated PR to remove HTML for nearly all facility service export displays (the one exception being the vet center services description field)

File: config/sync/views.view.facility_services.yml [hunk 1]
```diff
@@ -10625,20 +10731,21 @@ display:
...
-          plugin_id: entity_link_edit
-          label: ''
+          entity_field: nid
+          plugin_id: field
+          label: URL
           exclude: false
           alter:
             alter_text: true
-            text: '<a href="/node/{{ nid }}/edit" class="button">Edit</a>'
+            text: '/node/{{ nid }}'
```
Comment (swirtSJW) [inline config/sync/views.view.facility_services.yml:10748]:
```suggestion
            text: 'https://prod.cms.va.gov/node/{{ nid }}'
```
Lets give them a fully qualified link to play with.

File: config/sync/views.view.facility_services.yml [hunk 2]
```diff
@@ -13636,20 +14102,21 @@ display:
...
```
Comment (swirtSJW) [inline config/sync/views.view.facility_services.yml:14119]:
```suggestion
            text: 'https://prod.cms.va.gov/node/{{ nid }}'
```

File: config/sync/views.view.facility_services.yml [hunk 3]
```diff
@@ -18417,20 +19306,21 @@ display:
...
```
Comment (swirtSJW) [inline config/sync/views.view.facility_services.yml:19323]:
```suggestion
            text: 'https://prod.cms.va.gov/node/{{ nid }}'
```

**Review by swirtSJW — COMMENTED** [review]
Just a minor tweak.

**Review by swirtSJW — APPROVED** [review]
Thanks Jay.

**Pull request merged**